### ISSUE TITLE
AnimePahe Not Playing Video In Aniyomi But Working Fine In Webview On Jio (ISP)

### ISSUE THREAD

**Issue by YouSummoner** [aniyomiorg/aniyomi-extensions#3256]
### Source information

AnimePahe 14.27

### Source language

English 

### Steps to reproduce

1. Try to play video or download on AnimePahe Source (won't work)
2. Open webview and try playing in that (will work)

### Expected behavior

The episode should play without issue.

### Actual behavior

For some reason, the source can't play episodes in Aniyomi on Jio Network (ISP) specifically, but when trying to open the AnimePahe website in webview or in a browser on the same network, it works and plays the video without any issue. Not sure what's happening but it's clear that the AnimePahe website isn't even blocked on Jio, it's some issues with the extension itself.

I've attached the logs from Aniyomi and a screenshot of AnimePahe working fine and playing the episode without issue in the app's webview on the same ISP.

### Aniyomi version

Preview r7540

### Android version

Android 14

### Other details

![Screenshot_20240521-211523_Aniyomi](https://github.com/aniyomiorg/aniyomi-extensions/assets/107224555/a1ddad8b-3b1c-49b5-a811-249ba178b602)
[aniyomi_crash_logs.txt](https://github.com/aniyomiorg/aniyomi-extensions/files/15392491/aniyomi_crash_logs.txt)
[aniyomi_crash_logs2.txt](https://github.com/aniyomiorg/aniyomi-extensions/files/15392492/aniyomi_crash_logs2.txt)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.2.4](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/docs/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/aniyomiorg/aniyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
